stdlib/public/runtime/CMakeLists.txt

```diff
@@ -148,6 +123,30 @@ add_swift_target_library(swiftImageRegistrationObjectCOFF
                   INSTALL_IN_COMPONENT none)
 
 foreach(sdk ${SWIFT_SDKS})
+  if(SWIFT_BUILD_STATIC_STDLIB AND "${SWIFT_SDK_${sdk}_OBJECT_FORMAT}" STREQUAL "ELF")
```

Comment:
Rather than only generate this link file if the host SDK is linux, as was done above, create it for all ELF SDKs in the list of wanted `SWIFT_SDKS`.